gprofiler/metadata/system_metadata.py

```diff
@@ -142,6 +147,10 @@ def get_cpu_info() -> Tuple[str, str]:
     Parse /proc/cpuinfo to get model name & flags.
     """
     try:
+        if is_windows():
```

Comment:
This function returns flags in a format specific for Linux, so I don't want to add Windows yet which creates a different format. I suggest we don't call it in Linux at all (use unknown value for both)

Reply:
Yeah I understand, that's what I meant - the function, when runs on Linux, returns data in very specific format, and the Windows values returned by your implementation will probably be very different and will mix it up... So I prefer we just use it at all on Windows, i.e `return UNKNOWN_VALUE, UNKNOWN_VALUE` if Windows.

I now notice that my comment was "we don't call it in Linux at all", I meant Windows, sorry :sweat_smile:

Reply:
Done